### ISSUE TITLE
Confirm if Meetecho is no longer providing slide names.

### ISSUE THREAD

**Issue by danyork** [stpeter/chatroom-relay#2]
In the draft, we say:

```
o  Determine if the session will be streamed via a real-collaboration
      system such as MeetEcho.  If so, that system might automatically
      post the slide names into the chatroom and this is one less task
      you need to be concerned about.
```

However, at IETF 91 it appears that slide titles are _NOT_ appearing in the chat rooms.  I suspect that they appeared in chat rooms before because the human Meetecho monitors in each room were manually adding the slide titles.  Now that Meetecho is centrally managing streams out of all rooms they no longer have human monitors in the room. 

We should confirm whether this is indeed the case... in which case this suggestion can be deleted.

**Comment by stpeter** [comment]
I've asked the Meetecho team to confirm.

**Comment by lminiero** [comment]
Hi Peter and Dan,

yes, this is something we used to do but they went "away" for some changes we did recently to improve scalability.

Specifically, in the past we preloaded all slide decks to convert them in a web format, and then triggered slide changes manually through the web interface: this allowed us to show the current slide easily, and notify it via Jabber as well. At the latest IETF in Honolulu, though, we had to cover all 8 sessions in parallel, and so such an approach was not possible anymore due to a lack of the required manpower. To overcome this, the slides are now made available as a low framerate video feed coming directly from the beamer in the room, which we then "WebRTC-ize" through our Janus WebRTC gateway. This helped us take care of some well known issues in the old approach (animations being lost in the process, some poorly converted slides, slide decks missing or being changed on the fly by chairs and so on) but obviously had the side effect of making us loose the automated slide-to-jabber thing.

We did indeed start working on an automated application, some kind of a bot, that joins the Jabber room, processes the beamer feed we send around, compares it with slide decks preloaded the old way and, if it recognizes a slide change or trigger, notifies it via Jabber automatically. It kinda worked in some experiments we did in our lab, but at its current stage wasn't accurate enough for bringing it in the field, especially in a scenario as demanding as the IETF community. For instance, despite some optimizations and "tricks" we involved, it had a lot of trouble figuring out the right slide when they were not full screen, or scrolling, or half page, something that with the creative ways some chairs have to display decks is not at all an edge case :-). We eventually thought that no info was better than the wrong info, and so put that on ice. We're still working on improving this, as the idea is to hopefully make that stable and reliable enough for Dallas or Yokohama.

Hope that helps, nice holidays to both of you!
